docs/sources/flow/reference/components/otelcol.connector.host_info.md

```diff
@@ -0,0 +1,115 @@
+---
+aliases:
+  - /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.connector.host_info/
+  - /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.connector.host_info/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.host_info/
+description: Learn about otelcol.connector.host_info
+labels:
+  stage: experimental
+title: otelcol.connector.host_info
+---
+
+# otelcol.connector.host_info
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT_VERSION>" >}}
+
+`otel.connector.host_info` accepts span data from other `otelcol` components and generates usage metrics.
+
+## Usage
+
+```river
+otelcol.connector.host_info "LABEL" {
+  output {
```

Comment:
This output block is not documented for some reason? cc @jcreixell

Reply:
thanks for catching this @thampiotr. opened a pr to document this block: https://github.com/grafana/agent/pull/6488